### PR TITLE
Using same naming and tagging as other backend components

### PR DESCRIPTION
**Pull request by svenngjeto**

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ Metadata:
     Author: Unit
     SpdxLicenseId: MIT
     LicenseUrl: LICENSE
+    Labels: ['${GIT_REPO}', '${CODEBUILD_RESOLVED_SOURCE_VERSION}', '@${BUILD_TIMESTAMP}']
```
Comment (janbj) [inline template.yaml:15]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/0

Comment (janbj) [inline template.yaml:15]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/1

Comment (janbj) [inline template.yaml:15]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/2

Comment (svenngjeto) [reply template.yaml:15]: same as always: envsubst takes care of substitution environment variables

Comment (svenngjeto) [reply template.yaml:15]: same as always: envsubst takes care of substitution environment variables

Comment (svenngjeto) [reply template.yaml:15]: same as always: envsubst takes care of substitution environment variables

**Pull request merged**